src/Search.php

```diff
@@ -8757,7 +8757,7 @@ public static function showItem($type, $value, &$num, $row, $extraparam = '')
                     $value = preg_replace('/' . self::LBBR . '/', '<br>', $value);
                     $value = preg_replace('/' . self::LBHR . '/', '<hr>', $value);
                     $value = '<div class="fup-popup">' . $value . '</div>';
-                    $valTip = "&nbsp;" . Html::showToolTip(
+                    $valTip = '' . Html::showToolTip(
```

Comment:
I think this case is "valid" as it is used as a spacer and shouldn't ever be result in `&nbsp;` being returned on its own.